db/schema.rb

```diff
@@ -173,13 +173,13 @@
     t.integer  "primary_payment_method_id"
     t.string   "type",                      limit: 255
     t.integer  "current_cohort_id"
-    t.string   "invitation_token",          limit: 255
```

Comment:
I'll need some help with this. When I run an interactive rebase this commit is not present, even though it shows up here and in my git log.